x/wasm/keeper/gas_register.go

```diff
@@ -0,0 +1,117 @@
+package keeper
+
+import (
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// DefaultGasMultiplier is how many cosmwasm gas points = 1 sdk gas point
+// SDK reference costs can be found here: https://github.com/cosmos/cosmos-sdk/blob/02c6c9fafd58da88550ab4d7d494724a477c8a68/store/types/gas.go#L153-L164
+// A write at ~3000 gas and ~200us = 10 gas per us (microsecond) cpu/io
+// Rough timing have 88k gas at 90us, which is equal to 1k sdk gas... (one read)
+//
+// Please note that all gas prices returned to the wasmer engine should have this multiplied
+const DefaultGasMultiplier uint64 = 100
+
+// DefaultInstanceCost is how much SDK gas we charge each time we load a WASM instance.
+// Creating a new instance is costly, and this helps put a recursion limit to contracts calling contracts.
+const DefaultInstanceCost uint64 = 40_000
+
+// DefaultCompileCost is how much SDK gas we charge *per byte* for compiling WASM code.
+const DefaultCompileCost uint64 = 2
+
+// DefaultEventAttributeCost is how much SDK gas we charge *per byte* for attribute data in events
+const DefaultEventAttributeCost uint64 = 10
```

Comment:
Where does 10 come from?
Seems higher than IAVL, that's why I ask.

Reply:
Random value. I will change it to `1` as proposed by you. It is 10 here as I would have to update the tests as well ;-)